recipes-wpe/wpeframework/wpeframework-clientlibraries_git.bb

```diff
@@ -0,0 +1,75 @@
+SUMMARY = "WPEFramework client libraries"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=f1dffbfd5c2eb52e0302eb6296cc3711"
+PR = "r0"
+
+require include/wpeframework.inc
+
+SRC_URI = "git://github.com/rdkcentral/ThunderClientLibraries.git;protocol=git;branch=master"
+SRCREV = "bea2273b40525b01cc306e4acb03d13bf3d79313"
+
+# ----------------------------------------------------------------------------
+
+include include/compositor.inc
+
+DEPENDS = " \
+    wpeframework wpeframework-interfaces \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'compositor', '${WPE_COMPOSITOR_DEP}', '', d)} \
+"
+
+WPE_CDMI_ADAPTER_IMPL ?= "${@bb.utils.contains('DISTRO_FEATURES', 'nexus_svp', 'opencdmi_brcm_svp', 'opencdm_gst', d)}"
+
+PACKAGECONFIG ?= " \
+    release \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'opencdm', 'opencdm ${WPE_CDMI_ADAPTER_IMPL}', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'provisioning', 'provisionproxy', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'security', 'securityagent', '', d)} \
+    virtualinput \
+    "
+
+# Buildtype
+PACKAGECONFIG[debug]          = "-DBUILD_TYPE=Debug,,"
+PACKAGECONFIG[debugoptimized] = "-DBUILD_TYPE=DebugOptimized,,"
+PACKAGECONFIG[releasesymbols] = "-DBUILD_TYPE=ReleaseSymbols,,"
+PACKAGECONFIG[release]        = "-DBUILD_TYPE=Release,,"
+PACKAGECONFIG[production]     = "-DBUILD_TYPE=Production,,"
+
+PACKAGECONFIG_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'compositor', 'compositorclient', '', d)}"
+
+PACKAGECONFIG[compositorclient] = "-DCOMPOSITORCLIENT=ON,-DCOMPOSITORCLIENT=OFF"
+PACKAGECONFIG[provisionproxy]   = "-DPROVISIONPROXY=ON,-DPROVISIONPROXY=OFF,libprovision"
+PACKAGECONFIG[securityagent]    = "-DSECURITYAGENT=ON, -DSECURITYAGENT=OFF"
+PACKAGECONFIG[virtualinput]     = "-DVIRTUALINPUT=ON,-DVIRTUALINPUT=OFF,"
+
+# OCDM
+PACKAGECONFIG[opencdm]          = "-DCDMI=ON,-DCDMI=OFF,"
+PACKAGECONFIG[opencdm_gst]      = '-DCDMI_ADAPTER_IMPLEMENTATION="gstreamer",,gstreamer1.0'
+PACKAGECONFIG[opencdmi_brcm_svp]= '-DCDMI_BCM_NEXUS_SVP=ON -DCDMI_ADAPTER_IMPLEMENTATION="broadcom-svp",,gstreamer-plugins-soc'
+
+# ----------------------------------------------------------------------------
+
+EXTRA_OECMAKE += " \
+    -DBUILD_SHARED_LIBS=ON \
+    -DBUILD_REFERENCE=${SRCREV} \
+    -DPLUGIN_COMPOSITOR_IMPLEMENTATION=${WPE_COMPOSITOR_IMPL} \
+    -DPLUGIN_COMPOSITOR_SUB_IMPLEMENTATION=${WPE_COMPOSITOR_SUB_IMPL} \
+    -DPYTHON_EXECUTABLE=${STAGING_BINDIR_NATIVE}/python3-native/python3 \
```

Comment:
is PYTHON_EXECUTABLE required for clientlibs ?

Reply:
ocdm is looking for ProxyStubGenerator package that in-turn looks for PYTHON_EXECUTABLE.